### ISSUE TITLE
Text contrast

### ISSUE THREAD

**Issue by lukavdplas** [CentreForDigitalHumanities/cookiecutter-webapp-deluxe#52]
Bulma's CSS doesn't comply with WCAG by default, so applications should to make the following corrections for colour contrast:

- Add underlines to links, as there is no clear contrast between links and main text (the colour difference in black vs indigo is not clear enough)
- Increase colour contrast in primary / success / info / danger buttons (and various other elements that contrasts white with the primary/success/info/danger colours).
